components/calendar/src/iso.rs

```diff
@@ -302,6 +302,23 @@ impl Iso {
         }
     }
 
+    // Minute count representation of calendars starting from  00:00:00 UTC on Jan 1st, 1970.
+    pub fn unix_epoch_minute_from_iso(date: IsoDateInner) -> i32 {
+        let minutes_a_day = 60 * 24;
+        (Self::fixed_from_iso(date)
+            - Self::fixed_from_iso(IsoDateInner(ArithmeticDate::new(1969, 12, 31))))
+            * minutes_a_day
+    }
+
+    // Convert minute count since 00:00:00 UTC on Jan 1st, 1970 to ISO Date.
```

Comment:
Nit: Remove the reference to "UTC" since we are only talking in local datetimes here, not zoned datetimes.